src/ParallelDataCollector.cpp

```diff
@@ -1139,4 +1112,56 @@ namespace splash
         dataset.close();
     }
 
+    /* UNIMPLEMENTED METHODS FROM DATACOLLECTOR. TODO: Unify interface to remove those */
+    void ParallelDataCollector::readGlobalAttribute(const char*, void*, Dimensions*)
```

Comment:
Yes. This are the ones, that were not implemented before (empty, see the header: https://github.com/ComputationalRadiationPhysics/libSplash/pull/247/files/2bb3e900bb3aa67a2ef741ff3b2e98835e21ecd2#diff-786f550b56fdbb1aec6013d83cd5085fL360)